iNaturalist/src/main/res/values-ar/strings.xml

```diff
@@ -990,7 +990,7 @@
     <!-- Placeholder text shown in place of a username when the user has been deleted -->
     <string name="deleted_user">المستخدمين المحذوفين</string>
     <!-- Short date formatting template for dates in the current year. See https://developer.android.com/reference/java/text/SimpleDateFormat#date-and-time-patterns for formatting options -->
-    <string name="date_short_this_year">MMM d</string>
+    <string name="date_short_this_year">d MMM</string>
```

Comment:
These changes to the translation files aren't going to cause problems, but they are going to get overridden on the next Crowdin sync. IMO, we should leave them alone and rely on translators to make these changes.